src/io.c

```diff
@@ -24,6 +24,10 @@
 #define DISPATCH_IO_DEBUG DISPATCH_DEBUG
 #endif
 
+#ifndef PAGE_SIZE
+#define PAGE_SIZE 4096
```

Comment:
yup just checked, it's where it is, don't ask me how I can ever remember things like that...